jerry-core/vm/opcodes-helpers-variables.cpp

```diff
@@ -59,11 +59,10 @@ do_strict_eval_arguments_check (ecma_object_t *ref_base_lex_env_p, /**< base of
  *         false - otherwise.
  */
 bool
-is_reg_variable (vm_frame_ctx_t *frame_ctx_p, /**< interpreter context */
-                 vm_idx_t var_idx) /**< variable identifier */
+vm_is_reg_variable (vm_idx_t var_idx) /**< variable identifier */
```

Comment:
Is the method's documentation still valid? I mean there is a part where we refer to the "current interpreter context", but in the new implementation there is no context.